src/App.js

```diff
@@ -0,0 +1,17 @@
+import { Routes, Route } from 'react-router-dom';
+import NavBar from './components/NavBar';
+import BookPage from './Pages/BookPage';
+import CategoryPage from './Pages/CategoryPage';
+
+const App = () => (
+  <div className="App">
+     <NavBar/>
+     <Routes>
+       <Route path="/" element={<BookPage/>}/>
+       <Route path="/books" element={<BookPage/>}/>
```

Comment:
- [ ] Kindly remove this route, one route `/` which is the default route is enough for the book page 💯